ProjectManager/ProjectManager/Source/Protocol/MenuPresentable.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  MenuPresentable.swift
+//  ProjectManager
+//  Created by inho on 2023/01/18.
+//
+
+import UIKit
+
+protocol MenuPresentable: MainViewController {
+    func didLongPressGesture(_ sender: UIGestureRecognizer, _ viewModel: ListItemCellViewModel)
+}
+
+extension MenuPresentable {
+    func didLongPressGesture(_ sender: UIGestureRecognizer, _ viewModel: ListItemCellViewModel) {
+        guard sender.state == .began else { return }
+        
+        showPopoverMenu(sender, viewModel)
+    }
+    
+    func showPopoverMenu(_ sender: UIGestureRecognizer, _ viewModel: ListItemCellViewModel) {
+        let actionTypes = ListType.allCases.filter { $0 != viewModel.listType }
+        let menuAlert = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+            
+        menuAlert.modalPresentationStyle = .popover
+        menuAlert.popoverPresentationController?.permittedArrowDirections = [.up, .down]
+        menuAlert.popoverPresentationController?.sourceView = sender.view
+        
+        actionTypes.forEach {
+            let action = makeAlertAction(of: $0, viewModel: viewModel)
+            
+            menuAlert.addAction(action)
+        }
+        
+        present(menuAlert, animated: true)
+    }
+    
+    func makeAlertAction(of type: ListType, viewModel: ListItemCellViewModel) -> UIAlertAction {
```

Comment:
항상 뷰모델을 ListItemCellViewModel 만 주입받는다면,  아래처럼 숨겨줘도 되지 않을까요? 👀
```suggestion
    func makeAlertAction(of type: ListType, _ viewModel: ListItemCellViewModel) -> UIAlertAction {
```

Reply:
수정하였습니다..!
> 3a599de